spec/valkyrie/persistence/active_fedora/persister_spec.rb

```diff
@@ -4,5 +4,5 @@
 
 RSpec.describe Valkyrie::Persistence::ActiveFedora::Persister do
   let(:persister) { described_class }
-  it_behaves_like "a Valkyrie::Persister"
+  it_behaves_like "a Valkyrie::Persister", :no_deep_nesting, :no_mixed_nesting
```

Comment:
AF doesn't have an interface to support it in a generalized way. The raw Fedora adapter does support it, but that's not ready yet.